cypress/fixtures/categories_list.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "id": 1,
+    "name": "Politics"
+  },
+  {
+    "id": 2,
+    "name": "Opinion"
+  },
+  {
```

Comment:
dont we already have a fixture file for the `get` request for categories

Reply:
done ;)